astropy/coordinates/tests/test_funcs.py

```diff
@@ -33,6 +33,38 @@ def test_sun():
     gcrs2 = get_sun(Time([northern_summer_solstice, equinox_2, northern_winter_solstice]))
     assert np.all(np.abs(gcrs2.dec - [23.5, 0, -23.5]*u.deg) < 1*u.deg)
 
+def test_sun_02():
+    """
+    Test `that get_sun` produces similar values to skyfield (de421).
+
+    Commands to produce skyfield values:
+
+    from skyfield.api import sun, earth
+    from skyfield.units import Angle
+
+    apparent_gcrs = earth(utc=(2010,6,21)).observe(sun).apparent().radec()
+
+    skyf_ra_apparent = Angle(apparent_gcrs[0]).degrees
+    skyf_dec_apparent = Angle(apparent_gcrs[1]).degrees
+    skyf_dist_apparent = apparent_gcrs[2].AU
+    """
+    from ..funcs import get_sun
+
+    time_apy = Time('2010-6-21')
```

Comment:
Another minor nitpick comment:
I would remove the `_apy` everywhere. It's clear from the code that these are Astropy objects.
And keep the `skyf` or change to more explicit `skyfield` for the reference values.